addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -45,6 +45,20 @@ def _prepare_mail_values(self, res_ids):
                 )
                 if body:
                     mail_values['body_html'] = body
+            if mail_values.get('body'):
```

Comment:
:socks: commit header 'stop logging the full mailing ' is a bit misleading, isn't it ? It is more 'hide mailing content in logged note', something like that ?